Charts/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+*/
+!.gitignore
```

Comment:
Hm, I would add these to the  `.gitignore` in the project root.

Reply:
Okay! I needed to put something in there since otherwise git will ignore the directory, maybe I'll put a brief README instead.

Reply:
Grace, holler at me if you want to talk about this or have any questions.  (I'm a bit confused by git's behavior here.)

Reply:
FWIW, the `Charts/` directory ought to be created by the program if it doesn't exist, so I would actually just have `git` ignore the directory too.